pyaerocom/io/read_mscw_ctm.py

```diff
@@ -302,6 +308,8 @@ def read_var(self, var_name, ts_type=None, **kwargs):
             data.time.attrs['standard_name'] = 'time'
             data.attrs['units'] = self.preprocess_units(data.units, EMEP_prefix)
             cube = data.to_iris()
+            if ts_type == 'hourly':
+                cube.coords('time').convert_units('hours since 1900-01-01')
```

Comment:
I don't think `cube.coords('time')` works (cube.coords method returns a list I think), I think the syntax is `cube.coord('time')`. Please check that. Thanks.
```suggestion
                cube.coord('time').convert_units('hours since 1900-01-01')
```